### PR TITLE
Image Cleanup

### PR DESCRIPTION
**Pull request by dalex135**
@Nasser-Eddin-Nasser I changed the folder setup a bit. Please let me know if you don't like it.

### CODE REVIEW

File: Backend/src/main/java/Digitact/Backend/ConfigProperties.java [hunk 1]
```diff
@@ -7,7 +7,7 @@
 @Configuration
 public class ConfigProperties {
     public static final int serverPort = 9090;
-    public static final String absoluteFileSystemPath = "";
+    public static final String absoluteFileSystemPath = "D:\\Masters\\2nd Semester\\AMOS Project\\FileSystem\\";
```
Comment (Nasser-Eddin-Nasser) [inline Backend/src/main/java/Digitact/Backend/ConfigProperties.java]: can u remove the absoluteFileSystemPath  value.

File: Backend/src/main/java/Digitact/Backend/Controller/ClientController.java [hunk 2]
```diff
@@ -37,7 +37,7 @@
     public ResponseEntity<String> createApplicant(
             @RequestHeader HttpHeaders headers, @RequestBody ApplicantUI applicant) {
         Repository myRepos = new Repository(repository);
-        boolean isAuthorized = false;
+        boolean isAuthorized = true;
```
Comment (Nasser-Eddin-Nasser) [inline Backend/src/main/java/Digitact/Backend/Controller/ClientController.java]: should the default be false?

File: HR-Monitor/src/Controller/StandardController.java [hunk 3]
```diff
@@ -54,7 +54,7 @@ public StandardController(Stage stage) throws IOException {
         borderPaneCurrentView.setCenter(loadOverviewTableContent());
         textMenuLabel.setText(
                 IDictionary.getTranslation(menuDictionary, "Hello")
-                        + ": "
+                        + " "
```
Comment (Nasser-Eddin-Nasser) [inline HR-Monitor/src/Controller/StandardController.java:57]: good point, thanks :)

File: HR-Monitor/src/Main/App.java [hunk 4]
```diff
@@ -7,7 +7,7 @@
 
 public class App extends javafx.application.Application {
 
-    public static final boolean DEVELOPMENT_ENVIRONMENT = false;
+    public static final boolean DEVELOPMENT_ENVIRONMENT = true;
```
Comment (Nasser-Eddin-Nasser) [inline HR-Monitor/src/Main/App.java]: It should be false!

Comment (dalex135) [reply HR-Monitor/src/Main/App.java]: Yeah, this is corrected

**Review by vishwasanavatti — APPROVED** [review]
can be merged as it is just file name change.

**Pull request merged**